examples/kv/helm-chart/values.yaml

```diff
@@ -3,6 +3,7 @@
 ClusterEnv: dev
 ClusterName: examples-kv-cluster
 ClusterDomainName: cluster.local
+ClusterCookie: "tnt-kv-cookie"
```

Comment:
Tarantool Cartridge sets the default value of the cookie itself.
https://www.tarantool.io/ru/doc/latest/book/cartridge/cartridge_api/modules/cartridge/#cfg-opts-box-opts

Let's not change it unnecessarily.

Reply:
Agree with @vlsav it's more flexible.
For example, in my case I have docker image and use him in different projects, but have one password for all, think it's not secure.

Reply:
I mean, you shouldn't add this param to the default chart `values.yaml`.
in other words, you need to leave only what you fixed in `deployment.yaml`.

Reply:
adjected pull request according to feedback (in values.yaml line commented, with comment this is an example how to override cluster cookie definition)